cfme/tests/automate/custom_button/test_generic_objects.py

```diff
@@ -55,6 +56,22 @@ def setup_obj(appliance, button_group):
     return obj
 
 
+@pytest.fixture(scope="module")
+def method(custom_instance, button_group):
+    _, obj_type = button_group
+    obj_type = f"miq_{obj_type.lower()}" if obj_type == "GROUP" else obj_type.lower()
```

Comment:
it doesn't seem right to redefine the `obj_type` here

Reply:
I'm redefine it as per script demands. can you please explain me exact problem?

Reply:
Aha sorry. I didn't see that the object_type is in the RHS of the `=`.